### DISCUSSION TITLE
This looks really similar to `zk` and `zk-nvim`.

### DISCUSSION THREAD

**Discussion by dzintars** [artempyanykh/marksman#103] [General]
This thing at first glance looks similar to https://github.com/mickael-menu/zk and https://github.com/mickael-menu/zk-nvim which i use for some time.
For me the most important feature was "id based" file names which i found is only supported in Obsidian and ZK.
Posting this just for potential idea sharing. It's great there are innovation in this space.

**Comment by artempyanykh** [comment]
> This thing at first glance looks similar to https://github.com/mickael-menu/zk and https://github.com/mickael-menu/zk-nvim which i use for some time.

Great minds think alike 😄 Initially, Marksman too started as a Zettelkasten-specific tool but then evolved into a more general LSP for Markdown.

> For me the most important feature was "id based" file names which i found is only supported in Obsidian and ZK.

I think you want https://github.com/artempyanykh/marksman/issues/53 This is actually something I want to work on really soon.
